### ISSUE TITLE
libavutil.a fails linking on Windows

### ISSUE THREAD

**Issue by stoermelder** [VCVRack/VCV-Recorder#19]
commit 16e70f8d98d642abd78bd0bc89c9c71b435d1f11
Building on Windows 10

```
g++ -o plugin.dll build/src/Recorder.cpp.o build/src/plugin.cpp.o dep/lib/libavformat.a dep/lib/libavcodec.a dep/lib/libavutil.a dep/lib/libswscale.a dep/lib/libmp3lame.a -shared -L../.. -lRack
build/src/Recorder.cpp.o: In function `RecorderWidget::draw(rack::widget::Widget::DrawArgs const&)':
C:\msys64\home\User\Rackv1\plugins\VCV-Recorder/src/Recorder.cpp:1071: undefined reference to `glReadPixels'
dep/lib/libavutil.a(random_seed.o): In function `av_get_random_seed':
C:\msys64\home\User\Rackv1\plugins\VCV-Recorder\dep\ffmpeg/libavutil/random_seed.c:126: undefined reference to `BCryptOpenAlgorithmProvider'
C:\msys64\home\User\Rackv1\plugins\VCV-Recorder\dep\ffmpeg/libavutil/random_seed.c:129: undefined reference to `BCryptGenRandom'
C:\msys64\home\User\Rackv1\plugins\VCV-Recorder\dep\ffmpeg/libavutil/random_seed.c:130: undefined reference to `BCryptCloseAlgorithmProvider'
collect2.exe: error: ld returned 1 exit status
```

**Comment by Coirt** [comment]
Not sure if related but got a warning about library detection may fail when building, with the `pacman -Su mingw-w64-x86_64-lame` fix from #17. 
A fair few deprecated warnings in libavformat/codec, but dep still built okay. I built on one of the previous commits, not sure which commit 81ddaf7 or 9ca2a4b.

Fails on make without the fixes there obviously!

**Comment by stoermelder** [comment]
```
ifdef ARCH_WIN
	LDFLAGS += -lbcrypt
endif
```
Adding this to Makefile fixes the linking-error.
Still the reference to `glReadPixels` is missing.

**Comment by stoermelder** [comment]
```
ifdef ARCH_WIN
	LDFLAGS += -lbcrypt -lopengl32
endif
```
Builds successful :-)

**Comment by SmallTownSound** [comment]
First tests not good, Don't know if this actually works other than tricking compiler.     Trying video gives error 127.  All the audio formats work correctly. 
So we don't know which fix is the problem, since the original problem dealt with the ffmpeg....

**Comment by stoermelder** [comment]
Actually, recording video works, but only when the Rack-window is resized to the smallest possible window-size.

**Comment by SmallTownSound** [comment]
> 
> 
> Actually, recording video works, but only when the Rack-window is resized to the smallest possible window-size.

Unfortunately you are correct.  At least it is working now...

**Comment by stoermelder** [comment]
Fixed in ecc63a8a5cff07cc4feb367990dbab2127a7287c
